venue/views.py

```diff
@@ -1190,6 +1202,15 @@ def confirm_email_change(request):
         token = AuthToken.objects.get(salt=code)
         new_email = rtemp.retrieve(code)
         if code and new_email:
+            # Update the email in Constant Contact
```

Comment:
I think all this code should be done after checking if the user exists. If User.DoesNotExist error is thrown, then no update should occur in Constant Contact.